nss-day-cohort-47/tabloidfullstack-fiddlesticks
# Bk create tags

# Create a tag feature is complete

## Type of change


# Testing Instructions

Login and select tag management from the nav bar, click 'create a tag' and submit. View should reroute back to tags list and show new tag

# Checklist:
